### DISCUSSION TITLE
should we change all the code to be copy paste

### DISCUSSION THREAD

**Discussion by tomap** [hexojs/hexo#5480] [Question / Answer]
See https://github.com/hexojs/hexo-generator-feed/issues/211
Should we drop the $ sign
Cc @hexojs/core

**Comment by stevenjoezhang** [comment]
Indeed, removing the `$` symbol would make it more convenient for users to copy commands.

**Comment by uiolee** [comment]
markdownlint also recommends that don’t use dollar signs before commands without showing output.

https://github.com/DavidAnson/markdownlint/blob/main/doc/md014.md

**Comment by tomap** [comment]
Done for this repo :)
https://github.com/hexojs/hexo-generator-feed/pull/213
Maybe markdownlint to be enabled on the main repos ?  (at least on the mono repo)

**Reply by stevenjoezhang** [reply]
We can start with the https://github.com/hexojs/site/ repository, since it contains a lot of markdown files using `$` symbols in code blocks.
